### PR TITLE
Rename floating-point Condition Code

### PR DESCRIPTION
**Pull request by zhouyanggao**
Floating-point and fixed-point have the same condition code. So we
can use a more general name for the condition code.

Issue: #401
Signed-off-by: Zhouyang Gao <zhouyang@ca.ibm.com>

### CODE REVIEW

**Comment by zhouyanggao** [comment]
@ymanton  Since CR_LT, CR_GT, CR_EQ and CR_SO are defined in `/usr/include/sys/machine.h` on AIX, I am using CRCC_LT, CRCC_GT, CRCC_EQ and CRCC_SO instead. (CRCC stands for condition register condition code.) So the full name would be something like TR::RealRegister::CRCC_GT which I think is better than scoped enum TR::RealRegister::CRCC::GT.

File: compiler/p/codegen/OMRRealRegister.hpp [hunk 1]
```diff
@@ -78,12 +78,13 @@ class OMR_EXTENSIBLE RealRegister : public OMR::RealRegister
...
-      FPCC_FG = 0x1, // greater than
-      FPCC_FE = 0x2, // equal
-      FPCC_FU = 0x3  // unordered
-   } PPCFloatingPointConditionCode;
+   enum CRCC {
+      //Condition Register Condition Code (CRCC)
+      CRCC_LT = 0x0, // less than
+      CRCC_GT = 0x1, // greater than
+      CRCC_EQ = 0x2, // equal
+      CRCC_SO = 0x3  // summary overflow
```
Comment (ymanton) [inline compiler/p/codegen/OMRRealRegister.hpp]: What do you think about adding an alias for `SO` to represent `FU` and using that when dealing with FP regs? Something like:

```
enum CRCC {
       //Condition Register Condition Code (CRCC)
       CRCC_LT = 0x0,    // less than
       CRCC_GT = 0x1,    // greater than
       CRCC_EQ = 0x2,    // equal
       CRCC_SO = 0x3,    // summary overflow
       CRCC_FU = CRCC_SO // floating-point unordered
};
```

That would make the intention of the code clearer to anyone reading it if they didn't already know that the `SO` and `FU` bits were the same.

**Comment by vijaysun-omr** [comment]
@ymanton do you approve these changes ?

**Comment by ymanton** [comment]
Looks good.

**Pull request merged**